sass/type/_font-vars.scss

```diff
@@ -25,12 +31,11 @@
 ///    )
 /// );
 /// $font-var-prefix: 'prefix_';
-/// html { @include font--('my-font') }
+/// html { @include tools.font--('my-font') }
 ///
 /// @type string
 $font-var-prefix: 'font-' !default;
```

Comment:
```suggestion
tools.$font-var-prefix: 'font-' !default;
```

Reply:
This is the initial definition of the variable, so should not be prefixed.